src/frontend/staff-portal/src/app/components/staff-workbench/update-request-info/address-update-request-info/address-update-request-info.component.html

```diff
@@ -15,68 +15,68 @@
     <span class="section-grid-cell">
       <p class="section-grid-header">Requested Update</p>
     </span>
-    <span class="section-grid-cell" *ngIf="updateRequested.address_line1">
+    <span class="section-grid-cell" *ngIf="updateRequested.AddressLine1">
```

Comment:
Does the Staff API not use camel case for serialized members?

Reply:
these particular field names must be exactly the same as the DB names, but otherwise yes

Reply:
I think the Staff API does not serialize the object. Add models and use AutoMapper should be able to solve this

Reply:
how critical is this change?

Reply:
The API serializer can be configured with a naming convention. However, since  this is already in setup, no value at this point to making sweeping changes.